src/services/business.services.ts

```diff
@@ -9,23 +8,14 @@ export default class BusinessServices {
   /** Legal API URL. */
   static legalApiUrl = sessionStorage.getItem('LEGAL_API_URL')
 
-  /** Create a draft Incorporation Application for org and requested business */
-  static async createBusiness (filingBody: BusinessRequest): Promise<any> {
-    try {
-      const response = await axios.post(
-        `${BusinessServices.legalApiUrl}/businesses?draft=true`, filingBody
-      )
-      if (response?.status >= 200 && response?.status < 300) {
-        return response
-      }
-    } catch {
-      // delete the created affiliation if the update failed for avoiding orphan records
```

Comment:
It is not this method's job to clean up for something that happened in the parent. Hence, this is moved to the caller.